supervisor/docker/addon.py

```diff
@@ -276,6 +276,10 @@ def ulimits(self) -> Optional[List[docker.types.Ulimit]]:
     @property
     def cpu_rt_runtime(self) -> Optional[int]:
         """Limit CPU real-time runtime in microseconds."""
+        if not self.sys_docker.info.support_cpu_realtime:
```

Comment:
Should we log this happens? As in, the creator expected this to work maybe?

Reply:
It would work because it's disabled. Only HAOS have the current problem with the wrong implementation